### DISCUSSION TITLE
Prenses sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#267] [Zeka Sorusu]
Zeki, güzel ve yüksek adalet anlayışı ile tanınan bir prenses, kendisiyle evlenmek isteyen 3 zeki ve yakışıklı prens arasından en zeki olanını belirlemek istemektedir. Bu nedenle onlara bir zeka testi yapmaya karar verir.
3 prens de bir odaya konur ve yüzleri birbirine dönük şekilde sandalyelere oturtulur. Onlara, 2 adet siyah, 3 adet beyaz şapka gösterilir. Sonra gözleri, hiçbir şekilde göremeyecekleri şekilde bağlanır. Her birinin kafasına birer şapka takılır. Artan 2 şapka, odanın dışına saklanır.
Prenses, kafasındaki şapkanın rengini, şapkayı çıkarmadan ya da ona doğrudan bakmadan tahmin etmeyi başaran ilk kişiyle evleneceğini söyler. Ancak yanlış bir tahmin, derhal saraydan kovulma ve asla prenses ile görüşememe demektir. Gözlerindeki bağ, bu noktada açılır.
Siz, bu prenslerden birisiniz. Diğer iki prensin kafasına baktığınızda, ikisinde de beyaz şapka olduğunu görüyorsunuz. Tahmin etmeye çalışıyorsunuz; ama bir yolunu bulamıyorsunuz. Dahası, diğer prensler de tahmin edemiyor gibi gözüküyor.
Kafanızdaki şapka ne renk?
Not: Diğer iki prensin de en az sizin kadar zeki olduğunu ve en az sizin kadar prenses ile evlenmek istediğini biliyorsunuz. Dolayısıyla işbirliği yapma şansınız yok. Dahası, prensesin ne kadar zeki ve adil olduğundan da hiçbir kuşkunuz yok. Dolayısıyla bu testin gerçek bir mantık testi olduğundan eminsiniz.

**Comment by Denizmly** [comment]
Kafamdaki şapka beyaz renkte
Çünkü eğer pranses iki siyah şapkayıda kullansaydı beyaz şapka taktığı prense avantaj sağlanırdı. Ayrıca prenses adaletli ve zeki biri
